docs/metadata/prophecy-build-tool.md

```diff
@@ -78,7 +78,20 @@ be uploaded. These are the `--release-version` and `--project-id` parameters whi
 (`databricks-job.json`). Using a unique release version of your choice and the project's Prophecy ID
 (as seen in the project's URL on the Prophecy UI) is recommended.
 
-Example deploy command:
+##### Build command
+
+```shell
+pbt build --path /path/to/your/prophecy_project/
+```
+
+- we also have the ability to filter pipelines which we want to build, this can be huge time saving if we have large number of pipelines,
+- Additionally, multiple pipelines can be passed comma(,) separated.
```

Comment:
PBT provides user the ability to filter...
To only build certain pipelines we can use:
```code here```